app/views/spaces/spaces.erb

```diff
@@ -3,10 +3,10 @@
 <ul>
   <% @spaces.each do |space| %>
       <li>
-        Name: <%= space.name %>
-        Location: <%= space.address %>
-        Description: <%= space.description %>
-        Price per night: <%= space.price %>
+        Name: <a href="http://localhost:9292/spaces/<%=space.id%>" ><%= space.name %></a><br>
```

Comment:
shouldn't this be a relative route (/spaces/) - otherwise it won't work outside the test/dev environments